src/routes/+layout.svelte

```diff
@@ -59,7 +59,32 @@
         document.body.classList.add(className);
     }
 
-    onMount(() => {
+    onMount(async () => {
+        if (browser) {
+            const urlParams = $page.url.searchParams;
+            const ref = urlParams.get('ref');
+            const utmSource = urlParams.get('utm_source');
+            const utmMedium = urlParams.get('utm_medium');
+            const utmCampaign = urlParams.get('utm_campaign');
+            let referrer = document.referrer.length ? document.referrer : null;
+            // Skip our own
+            if(referrer?.includes('appwrite.io')) {
+                referrer = null;
+            }
```

Comment:
wouldnt this be useful here?

we have some subdomains leading back to our website